README.md

```diff
@@ -148,6 +148,30 @@ flagevalmm --tasks tasks/mmmu/mmmu_val.py \
 
 `--use-cache` is optional, it will cache the model outputs, the same question with the same model setting will get results from cache.
 
+## Start data server and evaluate seperately
+
+
+Above is one-key evaluation, you can also start data server seperately and evaluate seperately.Example of evaluating qwen-vl-2 model:
+
+```bash
+# Start data server
+python flagevalmm/server/run_server.py --tasks tasks/charxiv/charxiv_val.py --output-dir ./results/qwenvl2-7b --port 11823 
+```
+
+### Evaluate seperately
+
+This will start a server on port 11823, and the data server will be running until you stop it.
+```bash
+python flagevalmm/eval.py --output-dir ./results/qwenvl2-7b --tasks tasks/charxiv/charxiv_val.py --model your_model_path/Qwen2-VL-7B-Instruct/--exec model_zoo/vlm/qwen_vl/model_adapter.py
```

Comment:
- add a space between `your_model_path/Qwen2-VL-7B-Instruct/` and `--exec`
- add arg `--server-port 11823`